api/v1alpha1/nificonnection_types.go

```diff
@@ -102,7 +102,7 @@ func (nCon *NifiConnectionSpec) IsValid() bool {
 }
 
 func (ref *ComponentReference) IsValid() bool {
-	return ref.Type == ComponentDataflow && ref.SubName != ""
+	return ref.Type == ComponentDataflow && ref.SubName != "" || ref.Type == ComponentProcessGroup && ref.SubName != ""
```

Comment:
```suggestion
	return (ref.Type == ComponentDataflow && ref.SubName != "") || (ref.Type == ComponentProcessGroup && ref.SubName != "")
```

To be sure, things are evaluated correctly